packages/common/store/actions/urlProcessor.js

```diff
@@ -65,7 +64,7 @@ const setVotesByLaunchProtocol = search =>
 
     return dispatch(
       voteEdited(accounts.data
-        .filter(({ summary }) => validateAddress(tokenMap.LSK.key, summary.address) === 0)
+        .filter(({ summary }) => validateAddress(summary.address) === 0)
```

Comment:
it can return -1 or 1, from my perspective 0 is false but here actually means the address is valid, i think we need to return boolean on validateAddress

Reply:
Explained above.